qemu/tests/blockdev_mirror_hotunplug.py

```diff
@@ -1,15 +1,10 @@
 import time
 
 from virttest.qemu_monitor import QMPCmdError
-
```

Comment:
pls keep the current code style, you can submit another commit if you have concern about the code style